resources/views/layouts/nav.blade.php

```diff
@@ -21,6 +21,7 @@
             <ul class="nav navbar-nav">
                 <li><a href="{{ url('/home') }}">Home</a></li>
                 <li><a href="/jobs">Jobs</a></li>
+                <li><a href="/blog/about">About</a></li>
```

Comment:
You also might want to consider name routes so that if the route changes we don't have to go around updating all the routes.